### PR TITLE
LPS-161324 Remove SantizerUtil from modules

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina, 

This is resubmitted from #1482
I added the 7.4 version check in persistence_impl.ftl

### CODE REVIEW

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl [hunk 1]
```diff
@@ -923,8 +923,17 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 						<#else>
 							<#assign modes = "StringUtil.split(\"" + sanitizeTuple.getObject(2) + "\")" />
 						</#if>
+						<#if serviceBuilder.isVersionGTE_7_4_0() && dependencyInjectorDS>
```
Comment (julschong) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:926]: added checking if version is 7.4

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl [hunk 2]
```diff
@@ -2947,6 +2956,12 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 			</#if>
 			protected FinderCache finderCache;
 		</#if>
+		<#if serviceBuilder.isVersionGTE_7_4_0() && sanitizeTuples?size != 0>
```
Comment (julschong) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:2959]: added checking if version is 7.4

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl [hunk 3]
```diff
@@ -901,8 +901,8 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 				try {
 					<#list sanitizeTuples as sanitizeTuple>
 						<#assign
-							colMethodName = textFormatter.format(sanitizeTuple.getObject(0), 6)
-
+							colVariableName = sanitizeTuple.getObject(0)
+							colMethodName = textFormatter.format(colVariableName, 6)
```
Comment (tinatian) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:905]: @julschong You do not have version support here

**Comment by julschong** [comment]
will resend after adding version to <#assign>

**Pull request closed without merging**